OWML.Common/IModHelper.cs

```diff
@@ -14,5 +14,6 @@ public interface IModHelper
         IModManifest Manifest { get; }
         IModConfig Config { get; }
         IOwmlConfig OwmlConfig { get; }
+        IModInputHandler InputHandler { get; }
```

Comment:
I think it would be cool if the property is called Input, so it's used like this: ModHelper.Input...
The class/interfaces could still be called ModInputHandler

Reply:
sounds reasonable actually